packages/alchemy/src/middleware/gas-manager.ts

```diff
@@ -70,23 +35,40 @@ export const withAlchemyGasManager = <
         preVerificationGas: 0n,
         verificationGasLimit: 0n,
       }))
-      // no-op gas manager because the alchemy api will do it
+      // no-op fee because the alchemy api will do it
+      // can set this after the fact to do fee overrides.
       .withFeeDataGetter(async () => ({
         maxFeePerGas: 0n,
         maxPriorityFeePerGas: 0n,
       }))
       .withPaymasterMiddleware({
         paymasterDataMiddleware: async (struct) => {
+          const userOperation: UserOperationRequest = deepHexlify(
+            await resolveProperties(struct)
+          );
+
+          var feeOverride = undefined;
```

Comment:
use `let` if you want mutable or `const` if immutable. `var` is no bueno because it get's hoisted into global scope lol